migration/migrating-openshift-3-to-4.adoc

```diff
@@ -0,0 +1,92 @@
+[id="migrating-openshift-3-to-4"]
+= Migrating {product-title} version 3 to 4
+include::modules/common-attributes.adoc[]
+:context: migrating-openshift-3-to-4
+
+toc::[]
+
+You can migrate application workloads from {product-title} 3.7 to a later version with the Cluster Application Migration (CAM) tool. The CAM enables you to control the migration and to minimize application downtime. The CAM provides a web console and an API, based on Kubernetes custom resources, for migrating stateful application workloads, at the granularity of a namespace, from a source to a target cluster.
+
+You can migrate data to any storage class that is available on the target cluster, for example, from Red Hat Gluster Storage or NFS storage to Red Hat Ceph Storage.
+
+Optionally, you can migrate the {product-title} 3.7 (and later) control plane settings to {product-title} 4.x with the Control Plane Migration Assistant (CPMA). See xref:migration-understanding-cpma_{context}[].
+
+.Prerequisites
+
+* You must have `cluster-admin` privileges.
+* You must have `podman` installed.
+* The source cluster(s) must be {product-title} 3.7 or later.
+* The target cluster must be {product-title} 3.9 or later.
```

Comment:
Suggest we change to:

"The target cluster is recommended to be the latest released version of {product-title}."


For background...
This will technically work on 3.7 or later for both source and destination.
Recommendation is customer migrate to latest released OpenShift for the destination.

Reply:
I suggest we leave this as is. We should treat these situations with a support exception. 
This will be confusing if you look at this with the lifecycle.

Reply:
@sferich888 when you say leave as-is, I just want to confirm.

Right now doc says:  "The target cluster must be {product-title} 3.9 or later."

I think it should say:
  "The target cluster is recommended to be the latest released version of {product-title}."


Then we can address 3->3 migrations on a case-by-case basis.  Do you agree?